### DISCUSSION TITLE
Allow longer duration playlists

### DISCUSSION THREAD

**Discussion by peppy** [ppy/osu#14251] [Ideas] [answered]
To allow for more interesting usages of the playlists system, it would be nice to bump up the maximum duration to 3 months. Potentially supporter-only for durations above 2 weeks? Interested in hearing people's thoughts.

**Comment by venix12** [comment]
Hey, I wanted to ask if this could possibly be bumped in priority (and/or possibly implemented soon)? We wanted to give a try to the new spotlights model in which the lobbies would last 2 weeks. (we were planning to start it around early/mid january, so having it possible by then would be strongly appreciated)

**Reply by peppy** [reply]
2 weeks is already possible - are you looking to make rooms longer than this?

**Reply by Moltanicaa** [reply]
Two? I see only one
![osu_2021-12-08_13-34-59](https://user-images.githubusercontent.com/59089206/145193602-276f97ac-e63c-4f51-b78f-60d117092448.jpg)

**Reply by peppy** [reply]
Aha, my bad. I guess my opening post here is wrong.

**Comment by NotGumballer91** [comment]
I feel like 1 week itself is fine on it's own. I don't mind the extension.
But for supporter, How about indefinitely?

**Reply by peppy** [reply]
no

**Reply by NotGumballer91** [reply]
I know it sounds insane but I'm only implying it should last as their supporter lasts. That's just my thought

**Reply by frenzibyte** [reply]
Your suggestion didn't really show that it should last as one's supporter lasts. But regardless, I don't see a reason for indefinite playlists at all. It sounds like it's missing the point of what playlists is for, which is hosting leaderboards on a certain beatmap list for a period of time, not forever.

**Comment by peppy** [comment, marked as answer]
Will be available via #16024.
